doc/API.md

```diff
@@ -217,3 +216,36 @@ This command does not return anything for now.
 
 | Field          | Type      | Description                                          |
 | -------------- | --------- | ---------------------------------------------------- |
+
+### `gethistory`
+
+`gethistory` retrieves a paginated list of accounting events.
+
+Aiming at giving an accounting point of view, the amounts returned by this call are the total
+of inflows and outflows net of any change amount (that is technically a transaction output, but not a cash outflow).
+
+#### Request
+
+| Field         | Type         | Description                                                          |
+| ------------- | ------------ | -------------------------------------------------------------------- |
+| `start`       | int          | Timestamp of the beginning of the period to retrieve events for      |
+| `end`         | int          | Timestamp of the end of the period to retrieve events for            |
+| `limit`       | int          | Maximum number of events to retrieve                                 |
```

Comment:
What is the purpose of the limit? I would suspect it is to avoid an overload, but how would a caller retrieve the next events then?

To illustrate my point: if i'm putting a limit on `1000`, and i get `1000` events as a response. It hints that there may be actually more events but i was limited by the `limit` i've set. But then to get the rest of the elements i have to call it with a higher limit... Which kill the purpose of having a limit in the first place?

Reply:
Thinking more about this: if we ensure that all events are ordered by date, if there is a number of entries equal to the limit they can just request again with the timestamp of the last entry as the `start` parameter.